DefInjected/DamageDef/Damages_Environmental.xml

```diff
@@ -4,21 +4,21 @@
   <!-- EN: burn -->
   <Burn.label>oparzenie</Burn.label>
   <!-- EN: {0} has burned to death. -->
-  <Burn.deathMessage>{0} umarł/a przez oparzenia.</Burn.deathMessage>
+  <Burn.deathMessage>{0} {0_gender ? umarł : umarła} od oparzeń.</Burn.deathMessage>
   
   <!-- EN: flame -->
   <Flame.label>ogień</Flame.label>
   <!-- EN: {0} has burned to death. -->
-  <Flame.deathMessage>{0} umarł/a poprzez spalenie</Flame.deathMessage>
+  <Flame.deathMessage>{0} {0_gender ? spłonął : spłonęła}.</Flame.deathMessage>
   
   <!-- EN: frostbite -->
   <Frostbite.label>odmrożenie</Frostbite.label>
   <!-- EN: {0} has succumbed to frostbite. -->
-  <Frostbite.deathMessage>{0} umarł/a przez odmrożenia.</Frostbite.deathMessage>
+  <Frostbite.deathMessage>{0} {0_gender ? umarł : umarła} od odmrożeń.</Frostbite.deathMessage>
   
   <!-- EN: scratch -->
-  <TornadoScratch.label>zadraśnięcie</TornadoScratch.label>
+  <TornadoScratch.label>zadrapanie</TornadoScratch.label>
   <!-- EN: {0} has been killed by a tornado. -->
-  <TornadoScratch.deathMessage>{0} został zabity przez tornado.</TornadoScratch.deathMessage>
+  <TornadoScratch.deathMessage>{0} {0_gender ? został zabity : została zabita} przez tornado.</TornadoScratch.deathMessage>
```

Comment:
```suggestion
  <TornadoScratch.deathMessage>{0} został{0_gender ? : a} zabit{0_gender ? y : a} przez tornado.</TornadoScratch.deathMessage>
```

Reply:
Wyrażenia pomiędzy { } nie mogą zawierać spacji (gdzieś na forum Ludeon o tym czytałem). Jeśli się myle to mnie poprawcie :D